### PR TITLE
New package: Beancount

### PR DESCRIPTION
**Pull request by jppelteret**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-markdown2/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-markdown2/package.py:1]: Can you update copyright here and in the other packages:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
?

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by adamjstewart** [comment]
Please split this PR into separate PRs for each package, that makes it much easier for me to review. You can ping me on each PR and I'll start reviewing.

**Comment by adamjstewart** [comment]
Also see the latest docs at https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi

PyPI is preferred over GitHub if possible, and we have a new `pypi` attribute that replaces `homepage`/`url`/`list_url`

File: var/spack/repos/builtin/packages/py-markdown2/package.py (hunk 1 above)
Comment (jppelteret) [reply var/spack/repos/builtin/packages/py-markdown2/package.py:1]: Ok, will do. Thanks!

**Comment by jppelteret** [comment]
Thanks for letting me know about the `Pypi` attribute @adamjstewart. I'll adjust each package accordingly and will resubmit each in their own separate PRs.

**Pull request closed without merging**